### DISCUSSION TITLE
Using conversation chain with HuggingFace

### DISCUSSION THREAD

**Discussion by OCNAI** [FlowiseAI/Flowise#1493] [Q&A]
I've tried to create a simple conversation chain. I can easily get it to work with the OpenAI API, but whenever I try to use it with the ChatHuggingFace module I can't seem to get it to work.
I'm new to using Flowise and HuggingFace so this may seem like a stupid question, but I can't seem to find any answers elsewhere on the internet.
Why isn't it working? Am I using the wrong HuggingFace model? I've tried with several but none of them works. Can any of you maybe suggest a model I can use instead?
Am I supposed to use my write API key instead of read?
Am I doing something else wrong? 

I've attached a picture of the chatflow.
![Skærmbillede 2024-01-06 215029](https://github.com/FlowiseAI/Flowise/assets/155850614/98156d4f-f711-4dca-aaf3-44f1bfb5d6b1)

**Comment by HenryHengZJ** [comment]
This is because the prompt used for Conversation Chain is tailored to OpenAI ChatGPT. We'll be allowing user to connect Chat Prompt Template to Conversation Chain to allow much more customization. Most models on HuggingFace requires you to rewrite the prompt, as not every model works the same
